test/unit/api_spec.js

```diff
@@ -4488,6 +4488,27 @@ Caron Broadcasting, Inc., an Ohio corporation (“Lessee”).`)
       await loadingTask.destroy();
     });
 
+    it("gets operatorList, with marked content lang", async function () {
+      const loadingTask = getDocument(
+        buildGetDocumentParams("marked_content_lang.pdf")
+      );
+      const pdfDoc = await loadingTask.promise;
+      const pdfPage = await pdfDoc.getPage(1);
+
+      pdfDoc.annotationStorage.setValue("30R", { value: "test" });
+      pdfDoc.annotationStorage.setValue("31R", { value: true });
+
+      const opList = await pdfPage.getOperatorList({
+        annotationMode: AnnotationMode.DISABLE,
+      });
+      expect(opList.fnArray[0]).toEqual(OPS.beginMarkedContentProps);
```

Comment:
Why is it useful to get the lang parameter in the operator list which is used to draw the pdf ?